src/main/java/frc/robot/commands/ShooterHeld.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.CDSSubsystem;
+import frc.robot.subsystems.LimelightSubsystem;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class ShooterHeld extends CommandBase {
+  private ShooterSubsystem m_ShooterSubsystem;
+  private LimelightSubsystem m_LimelightSubsystem;
+  private CDSSubsystem m_CDSSubsystem;
+  private int i;
+  private boolean LLEnabled;
+
+  /** Creates a new ShooterHeld. */
+  public ShooterHeld(
+      ShooterSubsystem shooterSubsystem,
+      LimelightSubsystem limelightSubsystem,
+      CDSSubsystem cdsSubsystem,
```

Comment:
On the fence about whether it will be used. We might want to run the belt while the shooter is running, instead of having it manual, but there's also cases like #105.